cli/Cargo.toml

```diff
@@ -13,6 +13,8 @@ repository = { workspace = true }
 documentation = { workspace = true }
 keywords = { workspace = true }
 
+include = ["/LICENSE", "/examples/", "/src/", "/testing/", "/tests/"]
```

Comment:
I don't know the exact rule, but `build.rs` appears to be missing in `cargo package --list`.

Reply:
Good catch. Checking with`cargo package` also revealed that setting `include` here disables the default exclusion of `.*` files, so I had to add a few more to avoid including snapshot files from `insta`.